<launch>
  
    <node pkg="vslam_svo" type="good_feature" name="good_feature" clear_params="true" output="screen">

        <param name="cam_topic" value="/rgb"/>

        <!-- Camera calibration file -->
        <!-- rosparam file="$(find vslam_svo)/param/R200.yaml" /--> <!-- TUM1.yaml-->
        <!-- rosparam file="/home/davidz/work/data/sin2_tex2_h1_v8_d/camera_pinhole.yaml" /--> <!--camera_pinhole_R200 camera_atan-->

    </node>
  
  <node pkg="img_pub" type="test_pub_realsense" name="test_img_pub_realsense" output="screen">
      <param name="data_dir" value="/home/davidz/work/data/sin2_tex2_h1_v8_d/img"/>
      <param name="data_dir" value="/media/davidz/work/work/data/realsense/hotel"/>
      <param name="data_dir" value="/media/davidz/work/work/data/rgbd/rgbd_dataset_freiburg1_desk"/>
      <param name="data_dir" value="/media/davidz/work/work/data/rgbd/rgbd_dataset_freiburg1_desk2"/>
      <param name="data_dir" value="/media/davidz/work/work/data/rgbd/rgbd_dataset_freiburg2_desk"/>
      <param name="data_dir" value="/media/davidz/work/work/data/realsense/etasF4_06_01_01"/>
      <param name="data_dir" value="/home/davidz/work/data/ETAS_2F_640_30"/>
  </node>

</launch>
